pmpro-set-expiration-dates.php

```diff
@@ -298,8 +303,8 @@ function pmprosed_plugin_row_meta($links, $file)
 {
     if (strpos($file, 'pmpro-set-expiration-dates.php') !== false) {
         $set_links = array(
-            '<a href="' . esc_url('http://www.paidmembershipspro.com/add-ons/plugins-on-github/pmpro-expiration-date/') . '" title="' . esc_attr(__('View Documentation', 'pmpro')) . '">' . __('Docs', 'pmpro') . '</a>',
-            '<a href="' . esc_url('http://paidmembershipspro.com/support/') . '" title="' . esc_attr(__('Visit Customer Support Forum', 'pmpro')) . '">' . __('Support', 'pmpro') . '</a>',
+            '<a href="' . esc_url('http://www.paidmembershipspro.com/add-ons/plugins-on-github/pmpro-expiration-date/') . '" title="' . esc_attr(__('View Documentation', 'paid-memberships-pro')) . '">' . __('Docs', 'paid-memberships-pro') . '</a>',

```

Comment:
I think the text domain should be same as text-domain of the plugin and not PMPro core.

Reply:
update text domain to that of plugin 'pmpro-set-expiration-dates'